spec/order_total_spec.rb

```diff
@@ -0,0 +1,28 @@
+require 'order_total'
+
+describe OrderTotal do
+
+  let(:burger_description) {"Cheeseburger"}
+  let(:burger_price) {4.99}
+  let(:burger_to_s) {"#{burger_description}\t#{burger_price}"}
+  let(:burger) { double :dish, :description => burger_description,
+                              :price => burger_price,
+                              :to_s => burger_to_s }
+  let(:kebab_description) {"Kebab"}
+  let(:kebab_price) {3.00}
+  let(:kebab_to_s) {"#{kebab_description}\t#{kebab_price}"}
+  let(:kebab) { double :dish, :description => kebab_description,
+                              :price => kebab_price,
+                              :to_s => kebab_to_s }
+  let(:order_lines) {{burger =>  1, kebab =>  1}}
+  let(:order) {double :order, order_lines: order_lines}
+  subject(:order_total) {described_class.new}
+
+  context 'Calculating total' do
+    it {is_expected.to respond_to(:calculate).with(1).argument}
+
+    it 'calculates to total of the order' do
+      expect(order_total.calculate(order_lines)).to eq burger_price + kebab_price
```

Comment:
Metrics/LineLength: Line is too long. [81/80] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)